examples/scenarioExtendingBoom.py

```diff
@@ -0,0 +1,400 @@
+    translatingBody4.setIPntFc_F([[translatingBody4.getMass() / 12 * (
+                3 * (scGeometry.diameterArm / 2) ** 2 + scGeometry.heightArm ** 2), 0.0, 0.0],
+                                  [0.0, translatingBody4.getMass() / 12 * (scGeometry.diameterArm / 2) ** 2, 0.0],
+                                  [0.0, 0.0, translatingBody4.getMass() / 12 * (
+                                          3 * (scGeometry.diameterArm / 2) ** 2 + scGeometry.heightArm ** 2)]])
+    translatingBody4.setDCM_FP([[1.0, 0.0, 0.0], [0.0, 1.0, 0.0], [0.0, 0.0, 1.0]])
+    translatingBody4.setR_FcF_F([[0.0], [scGeometry.heightArm / 2], [0.0]])
+    translatingBody4.setR_F0P_P([[0], [0], [0]])
+    translatingBody4.setFHat_P([[0], [1], [0]])
+    translatingBody4.setRhoInit(0.0)
+    translatingBody4.setRhoDotInit(0.0)
+    translatingBody4.setC(400.0)
+    translatingBody4.setK(100.0)
+    translatingBodyEffector.addTranslatingBody(translatingBody4)
+
+    profiler4 = prescribedLinearTranslation.PrescribedLinearTranslation()
+    profiler4.ModelTag = "profiler"
+    profiler4.setTransAccelMax(0.0005)
+    profiler4.setTransPosInit(translatingBody4.getRhoInit())
+    profiler4.setSmoothingDuration(10)
+    scSim.AddModelToTask(dynTaskName, profiler4)
+    translatingBodyEffector.translatingBodyRefInMsgs[3].subscribeTo(profiler4.linearTranslationRigidBodyOutMsg)
+
+    translatingRigidBodyMsgData = messaging.LinearTranslationRigidBodyMsgPayload()
+    translatingRigidBodyMsgData.rho = scGeometry.heightArm  # [m]
+    translatingRigidBodyMsgData.rhoDot = 0  # [m/s]
+    translatingRigidBodyMsg4 = messaging.LinearTranslationRigidBodyMsg().write(translatingRigidBodyMsgData)
+    translatingRigidBodyMsg4.this.disown()
+    profiler4.linearTranslationRigidBodyInMsg.subscribeTo(translatingRigidBodyMsg4)
+
+    scLog = scObject.scStateOutMsg.recorder()
+    scSim.AddModelToTask(dynTaskName, scLog)
+    rhoLog = []
+    for outMsg in translatingBodyEffector.translatingBodyOutMsgs:
+        rhoLog.append(outMsg.recorder())
+        scSim.AddModelToTask(dynTaskName, rhoLog[-1])
+
+    if vizSupport.vizFound:
+        scBodyList = [scObject,
+                      ["arm1", translatingBodyEffector.translatingBodyConfigLogOutMsgs[0]],
+                      ["arm2", translatingBodyEffector.translatingBodyConfigLogOutMsgs[1]],
+                      ["arm3", translatingBodyEffector.translatingBodyConfigLogOutMsgs[2]],
+                      ["arm4", translatingBodyEffector.translatingBodyConfigLogOutMsgs[3]]]
+        viz = vizSupport.enableUnityVisualization(scSim, dynTaskName, scBodyList
+                                                  # , saveFile=fileName
+                                                  )
+        vizSupport.createCustomModel(viz
+                                     , simBodiesToModify=[scObject.ModelTag]
+                                     , modelPath="CUBE"
+                                     , color=vizSupport.toRGBA255("gold")
```

Comment:
see my commit to color each extendable boom component in a slightly different gray color, and give each successive element a slightly smaller diameter. Now Vizard makes the boom deployment of 4 segments much easier to see.

Reply:
Great change!